avro2tf/src/main/scala/com/linkedin/avro2tf/helpers/TensorizeInJobHelper.scala

```diff
@@ -230,7 +230,7 @@ object TensorizeInJobHelper {
   private def wrapStringArray: UserDefinedFunction = {
 
     udf {
-      stringSeq: Seq[String] => Seq(stringSeq)
```

Comment:
can you explain why this change is needed?

Reply:
There is a bug in the lib to dump TFRecord format. This is a workaround to TFRecord format for string types.